src/app/containers/RadioSchedule/Canonical/index.jsx

```diff
@@ -0,0 +1,130 @@
+import SectionLabel from '@bbc/psammead-section-label';
+import { Link } from '@bbc/psammead-story-promo';
+import { ServiceContext } from '#contexts/ServiceContext';
+import processRadioSchedule from '../utilities/processRadioSchedule';
+import webLogger from '#lib/logger.web';
+
+const logger = webLogger();
+
+const MarginWrapper = styled.div`
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    margin-top: ${GEL_SPACING_DBL};
+  }
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-top: ${GEL_SPACING_TRPL};
+  }
+  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    margin-bottom: ${GEL_SPACING_DBL};
+  }
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+`;
+
+const RadioScheduleSection = styled.section.attrs(() => ({
+  role: 'region',
+  'aria-labelledby': 'Radio-Schedule',
+}))``;
+
+const FrontPageRadioScheduleSection = styled(RadioScheduleSection)`
+  margin: 0 auto;
+  width: 100%; /* Needed for IE11 */
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN};
+  }
+`;
+
+const RadioScheduleSectionLabel = styled(SectionLabel)`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin-top: ${GEL_SPACING_QUAD};
+  }
+`;
+
+const RadioFrequencyLink = styled(Link)`
+  font-size: 14px;
+  line-height: 18px;
+`;
+
+const CanonicalRadioSchedule = ({ endpoint }) => {
+  const [schedule, setRadioSchedule] = useState();
+  const {
+    service,
+    script,
+    dir,
+    timezone,
+    locale,
+    radioSchedule: { header, frequenciesPageUrl, frequenciesPageLabel },
+  } = useContext(ServiceContext);
+
+  useEffect(() => {
+    const handleResponse = async response => {
+      const radioScheduleData = await response.json();
+
+      const schedules = processRadioSchedule(radioScheduleData, service);
+      setRadioSchedule(schedules);
+    };
+
+    const fetchRadioScheduleData = pathname =>
+      fetch(pathname, { mode: 'no-cors' })
+        .then(handleResponse)
+        .catch(e => logger.error(`HTTP Error: "${e}"`));
+
+    fetchRadioScheduleData(endpoint);
+  }, [endpoint, service, script, timezone, locale]);
+
+  if (!schedule) {
+    return null;
+  }
+
+  const renderFrequencyLink = (link, label) => (
+    <RadioFrequencyLink link={link}>{label}</RadioFrequencyLink>
+  );
+
+  return (
+    <FrontPageRadioScheduleSection>
+      <RadioScheduleSectionLabel
+        script={script}
+        labelId="Radio-Schedule"
+        service={service}
+        dir={dir}
+        bar={false}
+      >
+        {header}
+      </RadioScheduleSectionLabel>
+      <MarginWrapper>
+        <RadioSchedule
+          schedules={schedule}
+          locale={locale}
+          timezone={timezone}
+          script={script}
+          service={service}
+          dir={dir}
+        />
+        {frequenciesPageUrl &&
+          renderFrequencyLink(frequenciesPageUrl, frequenciesPageLabel)}
```

Comment:
```suggestion
          renderFrequencyLink()}
```

Reply:
Does this even need to be a function? We could just have something like

```
{frequenciesPageUrl && (
  <RadioFrequencyLink href={frequenciesPageUrl}>
    {frequenciesPageLabel}
  </RadioFrequencyLink>
)}
```

Reply:
Thanks, this suggestion has been effected!